.gitignore

```diff
@@ -24,3 +24,6 @@ src/test/data/sandbox/
 docs/_site/
 
 .vscode/
+
+/src/main/java/seedu/address/model/person/BlacklistStatus.java
```

Comment:
By any chance should this actually not be ignored? It sounds relevant to the code...

Reply:
I had this when `blacklistStatus` was an extra field, but now we are using it as an additional option in the `clientStatus` field, so I've kept the file in case we change to using it, but I don't wanna commit it because its not used.